### PR TITLE
Improves the search results.

### LINKED ISSUE #1088: Websearch results are not shown as in demo.
**Actual Behaviour**

Queries for search results only in single response.

**Expected Behaviour**

Queries for search should show a horizontal list of card tiles as shown in demo.

**Steps to reproduce it**

Ask susi to search for something e.g 'search for dog'.

**Screenshots of the issue**

image

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by saurabh30**
Fixes #1088 

Changes:

1. Redefined the websearch skill.Send a PR for that https://github.com/fossasia/susi_skill_data/pull/309

2.Changed http to https in duckduckgo url in API.js.

Demo Link:http://bewildered-print.surge.sh/

Screenshots for the change: 
image

### CODE REVIEW

**Comment by saurabh30** [comment]
@daminisatya @sayak119  Please review again.

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ class MessageSection extends Component {
    }
 
   componentDidUpdate() {
-    switch (this.state.currTheme) {
+      switch (this.state.currTheme) {
```
Comment (duttaditya18) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: Why this change?

Comment (saurabh30) [reply src/components/ChatApp/MessageSection/MessageSection.react.js]: I think these changes are shown because of rebase.

**Comment by saurabh30** [comment]
@duttaditya18  Remaining two commits are for just updating the branch as the pr was not reviewed for long.
